net-misc/youtube-viewer/metadata.xml

```diff
@@ -19,7 +19,8 @@
 	</upstream>
 	<use>
 		<flag name="gtk">Install the gtk GUI, called gtk-youtube-viewer (deprecated)</flag>
```

Comment:
Is this still used? Any chance to get rid of it? It looks weird having gtk, gtk2, gtk3 flags. Ie can you just rename it in older ebuilds if it installs gtk2-viewer on them?

Reply:
If it is deprecated, better not provide it at all. For an optional gui either use `gui` or `gtk` flag, regardless of GTK version.

Reply:
The `gtk` use flag is used on the current stable ebuild. My plan was to let these changes land in testing, then submit a STABLEREQ for 3.5.9-r1, finally followed by dropping the old stable version along with this tag.

However if you prefer I change the stable ebuild too, I can deprecate the gtk flag now.

Reply:
Hmm, it's messy but if you have a plan, sure...

Reply:
It is not in line with gnome proj policy for USE `gtk` though. It says use `gtk` for whatever preferred implementation of GTK there is.

Reply:
^ So what you want to do to "fix" this is use 'gtk' to reference gtk3, and 'gtk2' for gtk2. You can edit metadata.xml so 'gtk' flag in >3.7.0 means gtk3 and below are legacy. 
Check 
https://devmanual.gentoo.org/ebuild-writing/misc-files/metadata/index.html
https://www.gentoo.org/glep/glep-0068.html#package-restrictions
https://www.gentoo.org/glep/glep-0068.html#example-metadata-xml-file

I tested your package and it seems to be ok on that part.

Reply:
I've updated the metadata to reflect the USE flags correctly between versions and according to Gnome project guidelines.